actor/builtin/paymentbroker/paymentbroker.go

```diff
@@ -205,7 +207,9 @@ func (pb *Actor) CreateChannel(vmctx exec.VMContext, target address.Address, eol
 // target Redeem(200)          -> Payer: 1000, Target: 200, Channel: 800
 // target Close(500)           -> Payer: 1500, Target: 500, Channel: 0
 //
-func (pb *Actor) Redeem(vmctx exec.VMContext, payer address.Address, chid *types.ChannelID, amt *types.AttoFIL, validAt *types.BlockHeight, condition *types.Predicate, sig []byte) (uint8, error) {
+// If a condition is provided in the voucher, concatenate its params with supplied params to send a message.
+// Any non-fault error is considered a validation failure.
```

Comment:
`Any non-fault error when the condition is run is considered a validation failure.`